packages/flutter/lib/src/rendering/object.dart

```diff
@@ -2560,31 +2560,31 @@ abstract class RenderObject extends AbstractNode with DiagnosticableTreeMixin im
 
   /// Mark this node as needing an update to its semantics description.
   ///
-  /// The parameters [onlyLocalUpdates] and [noGeometry] tell the framework
+  /// The parameters `onlyLocalUpdates` and `noGeometry` tell the framework
   /// how much of the semantics have changed. Bigger changes (indicated by
   /// setting one or both parameters to false) are more expansive to compute.
   ///
-  /// [onlyLocalUpdates] should be set to true to reduce cost if the semantics
+  /// `onlyLocalUpdates` should be set to true to reduce cost if the semantics
   /// update does not in any way change the shape of the semantics tree (e.g.
   /// [SemanticsNode]s will neither be added/removed from the tree nor be moved
-  /// within the tree). In other words, with [onlyLocalChanges] the
```

Comment:
Aside: I'm surprised we don't get a warning in cases like these. I'm assuming there's no nearby symbol of that name defined? If not, this feels like something we should be able to catch in tooling.

Reply:
Well first of all we get tons of warnings from dartdocs, that's how I caught most of these issues. But we ignore them because there's THOUSANDS of false positives currently.

But secondly what we're doing here with arguments isn't what dartdoc currently expects anyway.

See https://github.com/dart-lang/dartdoc/issues/1475 and https://github.com/dart-lang/dartdoc/issues/1486.